### DISCUSSION TITLE
So when is lazer going to hit the AppStore

### DISCUSSION THREAD

**Discussion by Rain-0x01-39** [ppy/osu#28439] [Q&A] [answered]
> Our goal is to get the game on mobile app stores in early 2024.

**Comment by bdach** [comment, marked as answer]
When it's ready, and we can spare the minimum ~month long process of jumping through apple's hoops to get it approved on the store. [And it's not ready](https://github.com/ppy/osu/issues?q=is%3Aopen+is%3Aissue+label%3Aplatform%2FiOS) (https://github.com/ppy/osu/issues/24983 is especially egregious).
